codecov.yml

```diff
@@ -44,7 +45,9 @@ ignore:
   - "*.md"
   - "*.rst"
   - "**/*.pb.go"
+  - "**/*.pb.gw.go"
   - "types/*.pb.go"
+  - "types/*.pb.gw.go"
```

Comment:
Are these patterns (types and **) both required?

Reply:
No need, I will remove it